c7n/resources/ebs.py

```diff
@@ -128,6 +128,24 @@ def _filter_ami_snapshots(self, snapshots):
     return matches
 
 
+def _filter_kms_active(self, resources):
```

Comment:
Looks like this isn't used anymore.  Can you clean this up

Reply:
good catch thanks